src/libs/analytics/provider.ts

```diff
@@ -30,7 +32,8 @@ export const analytics: AnalyticsProvider = {
       amplitude.logEvent(eventName.amplitude, params)
     }
     if (await getIsMaestro()) {
-      const MOCK_ANALYTICS_SERVER_URL = 'http://10.0.2.2:4001' // NOSONAR(typescript:S5332) maestro is run locally, we don't use HTTPS
+      const host = deviceInfoModule.isEmulatorSync() ? '10.0.2.2' : 'localhost'
```

Comment:
Il faut déplacer le commentaire de Sonar `// NOSONAR(typescript:S5332) maestro is run locally, we don't use HTTPS` sur la ligne 35 comme tu as déplacé l'adresse IP hard codé ! Il faut bien vérifier Sonar ;)

Reply:
zut, j'ai pas regardé cette partie de sonar

il ne faut pas déplacer le commentaire, mais en ajouter un nouveau : c'est une autre alerte

Reply:
Oui c'est vrai que c'est une autre erreur ! J'ai lu rapidement :D

Reply:
t'as lu moins rapidement que moi =P